### ISSUE TITLE
Game Crash

### ISSUE THREAD

**Issue by laserjet25** [FTBTeam/FTB-Mods-Issues#74]
To help us help you, please provide as much information as you can, this helps us with debugging and reproducing the issue.

**Mod & Pack version**
FTB Chunks 1605.3.1-build.48
**Forge / Fabric version, if applicable**
Forge 36.2.4
**Modpack (with version), if applicable**
Monumental Experience 13.2
**If the issue is a crash; An external link to the crash report**
https://pastebin.com/bc8BuVwY
**How to reproduce the issue**
The game crashes after loading into a server. This does not happen on singleplayer worlds.

**Comment by asgsdfgrthtr** [comment]
latest version build 67
it seems to have happened when i was loading in the minimap, i was in spectator fastest speed flying around and it just crashed and now everytime i try load minimap with "M" it crashes. Minimap doesnt show up either, works in single player no problems, but on server my instance crashes, server stays up.

[crash-2021-10-17_23.59.30-client.txt](https://github.com/FTBTeam/FTB-Mods-Issues/files/7369756/crash-2021-10-17_23.59.30-client.txt)

[crash-2021-10-17_23.38.12-client.txt](https://github.com/FTBTeam/FTB-Mods-Issues/files/7369758/crash-2021-10-17_23.38.12-client.txt)

[crash-2021-10-17_23.27.28-client.txt](https://github.com/FTBTeam/FTB-Mods-Issues/files/7369761/crash-2021-10-17_23.27.28-client.txt)

first two differ in the crash error, but it happened from the same attempts to load minimap

**Comment by slave2midna** [comment]
Adding my crash logs to this, noticed it this morning after my server restarted - hadn't had any issues prior.  I found this thread with [Issue 89](https://github.com/FTBTeam/FTB-Mods-Issues/issues/89)

We are also using 1.16.5 FTB Chunks until some of our other mods can be finished for 1.18.2 then we do intend to move. In the meantime we did manage to fix the issue by deleteing the FTB Chunks folder, however if the issue is slated to return this cost all of our players their claimed land and they had to reclaim it.

Mild annoyance of course, we'll make due until a fix is back ported or we get to update! I just hope my crash logs help!

[crash-2022-05-28_16.09.53-client.txt](https://github.com/FTBTeam/FTB-Mods-Issues/files/8792205/crash-2022-05-28_16.09.53-client.txt)

[crash-withoutoptifine-client.txt](https://github.com/FTBTeam/FTB-Mods-Issues/files/8792209/crash-withoutoptifine-client.txt)
